### DISCUSSION TITLE
AWS IAM Role authentication ?

### DISCUSSION THREAD

**Discussion by NoamLiber-Reco** [postgresml/pgcat#837] [General]
Is there a support for authenticating between PgCat to RDS with IAM role instead of user/pass?

**Comment by chrisjowen** [comment]
I'm also interested in this, I tried creating a simple python script that periodically updates the config with the temp credentials but see timeouts and unable able to validate the connection pool.

The logging didn't show much more in debug level. I assume the admin username/password is an issue as RDS hides this from us.

**Reply by chrisjowen** [reply]
Ok this isn't a problem it seems, but RDS requires sslmode=require. 

Setting:

```
server_tls = true
verify_server_certificate = true
```

I now see in the logs a ciphersuite exchange but have error 

```
Client auth requested but no cert/sigscheme available 
```

Im not clear if the cert from Amazon is not in the keystore or the docker image doesn't have support for the cipher provided.
